.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -59,6 +59,9 @@ At a minimum, the following information should be added (but add more as needed)
 - [ ] <!---ci_exclude_tsan|msan|ubsan|coverage--> Exclude: All with TSAN, MSAN, UBSAN, Coverage
 - [ ] <!---ci_exclude_aarch64|release|debug--> Exclude: All with aarch64, release, debug
 ---
+- [ ] <!---ci_include_fuzzer--> Run only fuzzers related jobs (libFuzzer fuzzers, AST fuzzers, etc.)
```

Comment:
`ci_include_fuzzer` sounds like "run the standard tests and _additionally_ run fuzzers" but the description says "run _only_ fuzzers". This is slightly confusing.

Reply:
option's names predicates `ci_include_/ci_exclude_` are dictated how CI works - they automatically disable/enable corresponding jobs